README.md

```diff
@@ -1,103 +1,5 @@
-
-[venv]: http://pypi.python.org/pypi/virtualenv
-[wrapper]: http://www.doughellmann.com/projects/virtualenvwrapper/
-
-__note__
-
-If you do not use virtualenv, you can follow the instructions below to install
-the required Python libraries into your global Python installation.
-
-If you don't mind installing globally, just run
-
-    sudo easy_install pip
-
-followed by
-
-    sudo pip install -r requirements.txt
-
-__note__
-
-If you are running on Ubuntu/Debian you will need to first do
-
-    sudo apt-get install python-setuptools
-
-to install the required Python libraries.
-
-### Running tests locally
-
-You will need persona credentials for the site being tested. Get the URL being tested from mozwebqa.cfg in the project root, sign up for that site, and enter the credentials in a yaml file (see credentials.yaml in the project root). To avoid version control conflicts, you may want to store your credentials files separately from your source code.
-
-For all tests to pass, you will need to edit your MDN profile and ensure all fields are filled out.
-
-An example of running all non-destructive tests:
-
-    py.test --driver=firefox --credentials=/path/to/credentials/credentials.yaml
-
-An example of running all of the tests in one file:
-
-    py.test --driver=firefox --credentials=/path/to/credentials/credentials.yaml tests/test_sign_in.py
-
-An example of running one test in a file:
-
-    py.test --driver=firefox --credentials=/path/to/credentials/credentials.yaml tests/test_sign_in.py -k test_sign_out
-
-For more command line options see https://github.com/davehunt/pytest-mozwebqa or run
-
-    py.test --help
-
-Writing Tests
--------------
-
-If you want to get involved and add more tests then there's just a few things
-we'd like to ask you to do:
-
-1. Use the [template files][GitHub Templates] for all new tests and page objects
-2. Follow our simple [style guide][Style Guide]
-3. Fork this project with your own GitHub account
-4. Make sure all tests are passing, and submit a pull request with your changes
-
-[GitHub Templates]: https://github.com/mozilla/mozwebqa-test-templates
-[Style Guide]: https://wiki.mozilla.org/QA/Execution/Web_Testing/Docs/Automation/StyleGuide
-
-License
--------
-This software is licensed under the [MPL] 2.0:
-
-    This Source Code Form is subject to the terms of the Mozilla Public
-    License, v. 2.0. If a copy of the MPL was not distributed with this
-    file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-[MPL]: http://www.mozilla.org/MPL/2.0/
+This repository is no longer being actively maintained, and the contents has
+been removed. Please use the repository history to access the previous content.
+We are not currently executing or developing tests for MDN in this repository.
+You can read more about MDN development and testing in the
+[project documentation](https://kuma.readthedocs.org/en/latest/installation.html).
```

Comment:
I considered that, but instead decided to link to the top level. There are a couple of test related pages in the documentation and didn't want to deep link to just one (it also makes the link more fragile). That said, it appears I directly linked to the installation page, so I'll change it to simply https://kuma.readthedocs.org